postgresql/resource_postgresql_role.go

```diff
@@ -208,6 +209,12 @@ func resourcePostgreSQLRoleConfigurationParameter() *schema.Resource {
 				Required:    true,
 				Description: "Value of the configuration parameter",
 			},
+			roleParameterQuoteAttr: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Quote the parameter value as a literal",
+			},
```

Comment:
I played with using `SupressDiffFunc` in this definition to see if I could make Terraform just ignore the quote value, but that didn't seem to work. Can't say I fully understand the utility of that functionality.